decidim-core/db/migrate/20250221085317_add_user_type_to_action_logs.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class AddUserTypeToActionLogs < ActiveRecord::Migration[7.0]
+  class ActionLog < ApplicationRecord
+    self.table_name = :decidim_action_logs
+  end
+
+  def up
```

Comment:
I don't think we need separate `up` and `down` migrations here.

All these operations are reversable using a `change` migrations.